src/plugins/tts/plugin.tts.js

```diff
@@ -158,18 +158,35 @@ BookReader.prototype.initNavbar = (function (super_) {
 
       $el.find('.BRcontrols').prepend(this.refs.$BRReadAloudToolbar);
 
+      const renderVoiceOption = (language) => {
```

Comment:
Since this takes multiple voices as input, lets make this plural, and change the argument name here.
```suggestion
      const renderVoicesOptions = (voices) => {
```